shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/presentation/ui/DeletePhotoDialog.kt

```diff
@@ -0,0 +1,62 @@
+package com.epmedu.animeal.feeding.presentation.ui
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import com.epmedu.animeal.foundation.dialog.AnimealQuestionDialog
+import com.epmedu.animeal.foundation.preview.AnimealPreview
+import com.epmedu.animeal.foundation.theme.AnimealTheme
+import com.epmedu.animeal.resources.R
+
+@Composable
+fun DeletePhotoDialog(
+    photoItem: FeedingPhotoItem,
+    onConfirm: () -> Unit,
+    onDismiss: () -> Unit
+) {
+    val height = LocalConfiguration.current.screenHeightDp / 3
+    AnimealQuestionDialog(
+        title = stringResource(id = R.string.are_you_sure_you_want_delete_photo),
+        dismissText = stringResource(id = R.string.no),
+        acceptText = stringResource(id = R.string.yes),
+        onDismiss = onDismiss,
+        onConfirm = { onConfirm() },
+        content = {
+            Box(
+                modifier = Modifier.fillMaxWidth(),
+                contentAlignment = Alignment.Center
+            ) {
+                AsyncImage(
+                    modifier = Modifier
+                        .clip(RoundedCornerShape(16.dp))
+                        .height(height.dp),
+                    model = photoItem.uri,
+                    contentDescription = null,
+                    contentScale = ContentScale.Crop
+                )
+            }
+        }
+    )
+}
+
+@AnimealPreview
+@Composable
+fun DeletePhotoDialogPreview() {
```

Comment:
Can we reduce the padding between the photo and the buttons? It looks too big compared to Figma

Reply:
Regarding that design issue, my solution is based on the existing Dialogs. 
Since that dialog applies the styles as the default with default paddings, it is impossible to control it. 
see the `androidx.compose.material.AlertDialogContent`
![image](https://user-images.githubusercontent.com/7996136/235111554-f196649b-6955-4dc5-a552-7c4145e73507.png)


Anyway, here we have two scenarios, first- create a custom dialog by the design, second since we have a time pressure, use the existing and in the future, apply the custom dialog. 
I prefer the second scenario, what do you think? @Fileek

Reply:
Let's keep it as it is for now and create a ticket for this issue